pkgs/development/tools/csvq/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "csvq";
+  version = "1.17.11";
+
+  src = fetchFromGitHub {
+    owner = "mithrandie";
+    repo = "csvq";
+    rev = "v${version}";
+    sha256 = "sha256-cH4uUi/QhOxf9aXh3wRozSKPTd13PnxydKFoG32MIxM=";
+  };
+
+  vendorSha256 = "sha256-BIa5vAEIDNdD+xW9z7geyBJnSbOeXxCl0aHIuAMBIgE=";
+
+  doCheck = false;
```

Comment:
Do we need to disable the tests?

Reply:
I copied this from another Go package and forgot to remove this line.